plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java

```diff
@@ -80,6 +85,19 @@ public void run() {
         } catch (Exception e) {
             LOG.error("Failed to clean up inactive containers", e);
         }
+        logActiveContainers(activeContainers);
+    }
+
+    private void logActiveContainers(List<String> activeContainers) {
+        if (activeContainers.isEmpty()) {
+            LOG.info("No active containers were detected.");
+        } else {
+            StringBuilder activeContainersReport = new StringBuilder();
+            activeContainersReport.append(activeContainers.size() > 1 ? "Containers are registered" : "Container is registered");
```

Comment:
please simplify it. It is just for maintenance purposes so we don't have to sync message with amount of containers